assets/src/blocks/Gallery/GalleryFrontend.js

```diff
@@ -55,16 +37,11 @@ export const GalleryFrontend = ({
 
       {images.length ? (
         <>
-          {layout === 'slider' && <GalleryCarousel onImageClick={openLightbox} images={images} />}
-          {layout === 'three-columns' && <GalleryThreeColumns onImageClick={openLightbox} images={images} postType={postType} />}
-          {layout === 'grid' && <GalleryGrid onImageClick={openLightbox} images={images} />}
+          {layout === 'slider' && <GalleryCarousel images={images} expand={expand} />}
```

Comment:
Maybe we could create a `galleryBlock` constant `{images, expand}` and use this for all the styles? Would remove some duplications